lib/HTFeed/Stage/ImageRemediate.pm

```diff
@@ -116,6 +116,8 @@ sub remediate_image {
 	return $self->_remediate_tiff( $oldfile, @_ );
     }
 
+    $self->{jm}->inc("ImageRemediate_images");
```

Comment:
Could potentially add a label here (and elsewhere in imageremediate) based on the kind of image we are remediating (tiff vs. jpeg2000 vs. png vs. jpeg?)

Reply:
Added an issue for this. Resolving conversation.